exploration/0001-builtin-registry-capabilities.md

```diff
@@ -53,4 +57,4 @@ All conforming implementations of this specification MUST provide a set of <dfn>
 
 ## Alternatives Considered
 
-The alternative would be to not include any built-in functions, essentially outsourcing the task of curating and maintaining commonly used functions to the ecosystem. The details of how this could work out in practice are unclear.
+The alternative would be to not include any _built-in functions_, essentially outsourcing the task of curating and maintaining commonly used functions to the ecosystem.
```

Comment:
Could also list as an alternative the preceding form of this PR, which defines but does not require the interfaces.